### ISSUE TITLE
Synchronise the online skills

### ISSUE THREAD

**Issue by alok760** [fossasia/susi_installer#78]
The Smart Speaker uses its own SUSI_Server which uses the skills stored in the server itself(https://github.com/fossasia/susi_server) and the SUSI Skill Data - https://github.com/fossasia/susi_skill_data
Implement a system to synchronize skills.

**Comment by norbusan** [comment]
What do you mean with `synchronize`? Two-way synchronization is hard.
Downloading from upstream (`git pull`) is already happening in the normal update daemon
https://github.com/fossasia/susi_installer/blob/e6f86b96d10b8bd5a22ab53d4a5b3cad17489c34/raspi/update_daemon/update_check.sh#L53

**Comment by alok760** [comment]
@mariobehling can you please clarify on what is needed? You asked me to open an issue for this on Gitter. Do we just have to update the skills ?

**Comment by mariobehling** [comment]
We discussed this specific point in the meetings and decided that for public skills we only synchronize from SUSI.AI to the local device. Public skills should not be writable on the local device. A message should be displayed, if the user tries to edit them:

"You are trying to edit public Open Source skills on your local device. These skills are continuously developed together with the community on the online platform. Please login to [SUSI.AI](https://susi.ai) to edit public skills.

Hint Icon: Public skills are regularly updated on your device."

**Comment by mariobehling** [comment]
> Downloading from upstream (git pull) is already happening in the normal update daemon

This feature should also be visible to users. Furthermore it should be possible to trigger this feature in the UI.

**Comment by mariobehling** [comment]
I think the word synchronization could be misleading. Let's rather call this "Update" of skills on the local device.

**Comment by norbusan** [comment]
Agreed. Public skills are updated via the normal update daemon, and we could add a button somewhere to trigger an update, but private skills should be creatable and editable on the device.
